src/components/calculator/PrevalenceControls.tsx

```diff
@@ -256,16 +270,17 @@ export const PrevalenceControls: React.FunctionComponent<{
       <header id="location">
         <Trans>calculator.location_selector_header</Trans>
       </header>
-      <div className="form-group" hidden={isManualEntryCurrently}>
+      <div className="form-group">
         <ControlLabel
           id="top-location-typeahead"
           header={t('calculator.select_location_label')}
         />
         <Typeahead
           clearButton={true}
+          disabled={isManualEntryCurrently}
           highlightOnlyResult={true}
           id="top-location-typeahead"
-          inputProps={{ autoComplete: 'off' }}
+          inputProps={{ autoComplete: 'chrome-off' }}
```

Comment:
Just checking, does this stop autocomplete on other browsers?

Reply:
I checked firefox and chrome, but not any others.